src/phone-number-field.tsx

```diff
@@ -13,17 +13,44 @@ import {combineRefs} from './utils/common';
 const formatPhone = (regionCode: RegionCode, number: string): string =>
     formatAsYouType(number.replace(/[^\d+*#]/g, ''), regionCode);
 
-type Props = Omit<React.InputHTMLAttributes<HTMLInputElement>, 'value' | 'onInput'> & {
+type InputProps = Omit<React.InputHTMLAttributes<HTMLInputElement>, 'value' | 'onInput'> & {
     inputRef?: React.Ref<HTMLInputElement>;
     value?: string;
     defaultValue?: string;
     onInput?: (event: React.FormEvent<HTMLInputElement>) => void;
+    prefix?: string;
+    e164?: boolean;
```

Comment:
as you can see, I've added an optional `e164` prop that will return the E164 formatted number in the `onChangeValue` event